.github/MAINTAINERS_GUIDE.md

```diff
@@ -578,6 +578,18 @@ Testing in our CI setup uses changes to these files when creating test builds.
 Many good things come to an end. This can sometimes include commands and flags.
 When commands or flags need to be removed, follow these steps:
 
+<details>
+<summary>Deprecating features</summary>
+
+- Public functionality should be deprecated on the next `semver:major` version
+  - Add the comment `// DEPRECATED(semver:major): Description about the deprecation and migration path`
+  - Print a warning `PrintWarning("DEPRECATED: Description about the deprecation and migration path")`
+- Internal functionality can be deprecated anytime
+  - Add the comment `// DEPRECATED: Description about the deprecation and migration path`
+- Please add deprecation comments generously to help the person completely remove the feature and tests
```

Comment:
```suggestion
- Please add deprecation comments generously to help the next person completely remove the feature and tests
```

It wasn't so clear to who this was for 😉

Reply:
Whoops, good catch! 🐝 🐝 🐝